packages/react/src/hooks/usePersistentUserChoices.ts

```diff
@@ -0,0 +1,60 @@
+import type { UserChoices } from '@livekit/components-core';
+import { loadUserChoices, saveUserChoices } from '@livekit/components-core';
+import * as React from 'react';
+
+/**
+ * Options for the `usePersistentDeviceSettings` hook.
+ * @alpha
+ */
+export interface UsePersistentUserChoicesOptions {
+  /**
+   * The default value to use if reading from local storage returns no results or fails.
+   */
+  defaults?: Partial<UserChoices>;
+  /**
+   * Whether to prevent saving to persistent storage.
+   * @defaultValue false
+   */
+  preventSave?: boolean;
```

Comment:
wondering if `save` and `load` would make more sense than negating the option with `prevent*`

Reply:
I have the feeling that `preventSave=true` tells you a bit more than just `save=true`.

Reply:
What about `skipSave: boolean`?

Reply:
Really just wanted to hear your opinion on it, I think your argument is fair. Let's stick with `prevent*` then!